R/scope.R

```diff
@@ -368,9 +368,9 @@ renv_scope_trace <- function(what, tracer, envir = NULL) {
   call <- sys.call()
   call[[1L]] <- base::trace
   call[["print"]] <- FALSE
-  eval(call, envir = parent.frame())
+  suppressMessages(eval(call, envir = parent.frame()))
```

Comment:
Is this to suppress the `method`-related messages?

Reply:
It suppresses this message:

```R
trace(mean, quote(1+ 1))
#> Tracing function "mean" in package "base"
```